features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -0,0 +1,427 @@
+                                                    defaultMessage=' Develop'
+                                                />
+                                            </Typography>
+                                        </Grid>
+                                    </Grid>
+                                    {api.type !== 'WEBSUB' && (
+                                        <Grid
+                                            container
+                                            direction='row'
+                                            alignItems='left'
+                                            style={{ marginLeft: '15px' }}
+                                            justify='center'
+                                        >
+                                            <Grid item>
+                                                {isEndpointAvailable ? (
+                                                    <CheckIcon className={classes.iconTrue} />
+                                                ) : (
+                                                    <CloseIcon className={classes.iconFalse} />
+                                                )}
+                                            </Grid>
+                                            <Grid item style={{ marginBottom: '3px', marginLeft: '3px' }}>
+                                                <Typography variant='h7'>
+                                                    <FormattedMessage
+                                                        id='Apis.Details.Overview.CustomizedStepper.Endpoint'
+                                                        defaultMessage=' Endpoint'
+                                                    />
+                                                </Typography>
+                                            </Grid>
+                                            <Grid item>
+                                                <Link to={'/apis/' + api.id + '/endpoints'}>
+                                                    <LaunchIcon
+                                                        style={{ marginLeft: '10px' }}
+                                                        color='primary'
+                                                        fontSize='small'
+                                                    />
+                                                </Link>
+                                            </Grid>
+                                        </Grid>
+                                    )}
+                                    <Grid
+                                        container
+                                        direction='row'
+                                        alignItems='center'
+                                        style={{ marginLeft: '28px' }}
+                                        justify='center'
```

Comment:
For margins and paddings pls use MUI box

Reply:
Fixed